### ISSUE TITLE
Garbo going apeshit casting Sauce Contemplation

### ISSUE THREAD

**Issue by capncastom** [loathers/garbage-collector#2086]
noticed this happening in the past couple days.
Every few turns:

```
Executing Barf Turn/Barf
Casting Sauce Contemplation 158 times...
You acquire an effect: Saucemastery (790)
Sauce Contemplation was successfully cast.
Preference _concoctionDatabaseRefreshes changed from 1965 to 1966
```

such that at the end of a garbo session:

`Saucemastery (464189)`

That seems pretty wild. 
I suppose its not hurting anything, and my taffy summons are too expensive, but damn.

**Comment by horrible-little-slime** [comment]
Please share a full session log

**Comment by capncastom** [comment]
[capn_castom_20240909.txt](https://github.com/user-attachments/files/16937680/capn_castom_20240909.txt)

here it is, thanks for taking a look!

**Comment by capncastom** [comment]
you know. . . checking each of my session logs going backward from today, it looks like this started on Aug 31st.
On Aug 31st, I broke my routine and let autoscend do an Avant Guard run.
I would suppose maybe autoscend set some mp burning thing that never got cleared?
[capn_castom_20240831.txt](https://github.com/user-attachments/files/16937759/capn_castom_20240831.txt)

So perhaps not a Garbo issue at all?
Now to figure out how to undo it.
Sorry if I wasted your attention!
